group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -449,7 +454,7 @@ public void testJoiningNonExistingStreamsGroupNoMissingTopics() {
         assertEquals(Errors.NONE.code(), response.errorCode());
         assertFalse(response.memberId().isEmpty());
         assertEquals(1, response.memberEpoch());
-        assertTrue(response.activeTasks().isEmpty());
+        assertFalse(response.activeTasks().isEmpty());
```

Comment:
Updated this condition since the member id is present in the initial request, the client doesn't need to make a second request solely for task assignment.